docs/setup-and-installing.md

```diff
@@ -29,31 +29,35 @@ This guide explains how to set up the SpatialOS Unreal GDK and the Unreal Engine
 * Recommended storage: 15GB+ available space
 
 ### Network settings
-* Refer to the [SpatialOS network settings](https://docs.improbable.io/reference/latest/shared/get-started/requirements#network-settings) (SpatialOS documentation)
+* Refer to the [SpatialOS network settings](https://docs.improbable.io/reference/13.1/shared/get-started/requirements#network-settings) (SpatialOS documentation)
 
 ### Software
 To build the SpatialOS Unreal GDK module you need the following installed:
 * Windows 10, with Command Prompt or PowerShell as your terminal
 * [Git for Windows](https://gitforwindows.org)
-* [SpatialOS version 13](https://docs.improbable.io/reference/13.1/shared/get-started/setup/win)
+* [SpatialOS version 13.1](https://console.improbable.io/installer/download/stable/latest/win).
+This installs:<br>
+		- the [`spatial` CLI](https://docs.improbable.io/reference/13.1/shared/spatial-cli-introduction)<br>
+		- the SpatialOS [Launcher](https://docs.improbable.io/reference/13.1/shared/operate/launcher)<br>
+		- the 32-bit and 64-bit Visual C++ Redistributables
 * The [Windows SDK 8.1](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
 * Visual Studio [2015](https://visualstudio.microsoft.com/vs/older-downloads/) or [2017](https://visualstudio.microsoft.com/downloads/) (we recommend 2017)
 
-### Other
-As part of your Unreal GDK setup, you need to clone the SpatialOS fork of the Unreal Engine source code on GitHub. In order to get access to this fork, you need to link your GitHub account to a verified Epic Games account, and to have agreed to Epic's license. You will not be able to use the Unreal GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
+### Unreal Engine EULA
```

Comment:
From memory, we link to this section of the page from other places in the repo.
Will need a quick search and replace, I think?